hyperspy/learn/mva.py

```diff
@@ -185,8 +185,19 @@ def decomposition(self,
                     "normalize_poissonian_noise is set to False")
                 normalize_poissonian_noise = False
             if output_dimension is None:
-                raise ValueError("With the mlpca algorithm the "
-                                 "output_dimension must be expecified")
+                raise ValueError("With the MLPCA algorithm the "
+                                 "output_dimension must be specified")
+
+        if algorithm == 'orpca':
+            if normalize_poissonian_noise is True:
+                _logger.warning(
+                    "It makes no sense to do normalize_poissonian_noise with "
+                    "the ORPCA algorithm. Therefore, "
+                    "normalize_poissonian_noise is set to False")
+                normalize_poissonian_noise = False
```

Comment:
@thomasaarholt this line might make it clear - `normalize_poissonian_noise` doesn't make sense in ORPCA, because the whole point is that it can deal with Poisson noise robustly.